src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImpl.java

```diff
@@ -21,13 +25,44 @@
         List<SendResult> results = new ArrayList<>();
         if (!CollectionUtils.isEmpty(msgs)) {
             for (String msg : msgs) {
-                results.add(send(routingKey, msg));
+                results.add(sendMessage(routingKey, msg));
             }
         }
         return results;
     }
 
-    private SendResult send(String routingKey, String msg) {
+    public List<SendResult> send(String jsonContent, String routingKey) {
```

Comment:
Isn't the parameters swapped? Seems like the routing key is the first parameter in the interface. Not to mention that all other 'send' methods take the routing key as the first parameter...